source/connect/tableau.txt

```diff
@@ -29,6 +29,12 @@ Connect from Tableau without Authentication or TLS/SSL
 Connect from Tableau with the C Authentication Plugin
 -----------------------------------------------------
 
+.. note::
+
+   If running this tutorial on macOS, you must have ``sudo`` access to
+   set appropriate permissions on the MySQL Configuration file in step
+   2.
```

Comment:
This reads a little funny to me. Can we say something like:

"If you are running Tableau on macOS, you must have ``sudo`` access to set the appropriate permissions on the MySQL configuration file in step 2."

Also I think the 'c' in "Configuration" should be lowercase.  If you agree, change in the step's title too.